pubspec.yaml

```diff
@@ -1,10 +1,10 @@
 name: fff
-version: 1.0.1
+version: 2.0.0
```

Comment:
why you think to increase major version?

Reply:
I suggest to increase the minor part of the version: `1.1.0`, wdyt?

Reply:
This is the suggested way: https://dart.dev/null-safety/migration-guide#package-version